compiler/tflchef/proto/tflchef.proto

```diff
@@ -291,6 +291,10 @@ message BatchToSpaceNDOptions {
   // None
 }
 
+message BroadcastToOptions {
+  // None
```

Comment:
It seems that there's no available option for boradcast_to operation.
https://www.tensorflow.org/api_docs/python/tf/broadcast_to#args

Do we need this empty message type? If it is just for confirmation of no option, I'd like leave as comment.
Protobuf compiler will generate related code and data structure even for empty message ;(

Reply:
> Do we need this empty message type? 

I assume design of this protobuf is to explicitly add empty for empty option.

Reply:
I could build tflchef without any error after removing the empty message.

Reply:
> > Do we need this empty message type?
> 
> I assume design of this protobuf is to explicitly add empty for empty option.

I didn't know about that. But I just want to warn that unmecessery code generation 🤔
However, in current .proto, it seems that there're many missing options which is not used as like `BroadcastTo`.